test/test-functions

```diff
@@ -99,11 +103,18 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    PARAMS+="systemd.mask=systemd-networkd.service "
+    PARAMS+="rd.hostonly=0"
+else
+    PARAMS+="ro"
```

Comment:
`ro` appears to be used for testing `systemd-remount-fs.service`, so I'm not sure it should be removed.